### DISCUSSION TITLE
Angular Wizard Source Code

### DISCUSSION THREAD

**Discussion by pavan2610** [vmware/clarity#6339] [Q&A] [answered]
Hello I am looking for Wizard Source Code . I find only demo files in the following repo. 
https://github.com/vmware/clarity/tree/angular/src/app/wizard
Could you please share the link?
I need to implement custom wizard so thought to use the same code and modify the html . Will it be easier to do it?

**Comment by mathisscott** [comment, marked as answer]
For v5 and earlier, the wizard code can be found here: https://github.com/vmware/clarity/tree/v5/projects/angular/src/wizard

For v12 and later, the wizard code can be found here: https://github.com/vmware/clarity/tree/angular/projects/angular/src/wizard
